In-house patch to add the 'solariszones' and 'ldoms' hypervisor to Nova's list of
known hypervisors. This patch has not yet been submitted upstream.

--- nova-2015.1.2/nova/compute/hv_type.py.~1~	2015-10-13 07:52:44.000000000 -0700
+++ nova-2015.1.2/nova/compute/hv_type.py	2016-08-02 13:55:19.746264695 -0700
@@ -40,6 +40,8 @@
 PARALLELS = "parallels"
 PHYP = "phyp"
 QEMU = "qemu"
+SOLARISZONES = "solariszones"
+LDOMS = "ldoms"
 TEST = "test"
 UML = "uml"
 VBOX = "vbox"
@@ -61,6 +63,8 @@
     PARALLELS,
     PHYP,
     QEMU,
+    SOLARISZONES,
+    LDOMS, 
     TEST,
     UML,
     VBOX,
